en/concept/flight_modes.md

```diff
@@ -1,34 +1,36 @@
 # Flight Modes
 
-**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md).
+**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md), and are loosely grouped according to their level of control of the vehicle by the system vs. the user: "manual" modes, where the controller is essentially a pass-through from the user's remote control to the vehicle's actuators; "assisted" modes, where the system's outputs are coordinated with the user's commands; and, "auto" modes, where the controller requires little to no user input whatsoever.
```

Comment:
Agreed, good improvement! Very nice to have examples for each point.

I am quite new to PX4, and this new description really makes a lot of sense to me.